src/main/java/com/ibm/watson/developer_cloud/conversation/v1_experimental/model/MessageResponse.java

```diff
@@ -93,6 +94,24 @@ public void setLocation(Integer... location) {
     public void setValue(String value) {
       this.value = value;
     }
+
+    @Override public boolean equals(Object obj) {
```

Comment:
I had some junits that compared the intents returned with the expected model.. but now that you mention this I will just compare the two models